pages/index.tsx

```diff
@@ -0,0 +1,245 @@
+    const entryTime = new Date((entry.dt) * 1000).getHours();
+    return entryDate === date && entryTime >= 6;
+  });
+});
+
+//getting temperature array of next 5 days 
+const chart_temp = new Array<Chart>;
+firstDataForEachDate.map((d) => 
+  chart_temp.push({
+    temp:(Math.floor(d?.main.temp ?? 0)),
+    dates: (d ? format(parseISO(d.dt_txt), "dd.MM") : "")
+  }
+));
+
+return (
+  <div className="w-screen flex flex-col gap-4 justify-between bg-gray-100 overflow-scroll">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0"/> 
+    <nav className="w-screen flex shadow-sm justify-between sticky top-0 left-0 z-50 bg-white">
+        <div className="w-screen flex justify-between items-center max-w-7xl px-3 mx-auto">
+          <div className="text-4xl font-extrabold dark:text-white">Weather</div>
+          <MdWbSunny className="text-3xl mt-1 text-yellow-300" />
+          <section className="flex gap-2 items-center">
+            <MdMyLocation
+              title="Your Current Location"
+              className="text-2xl  text-gray-400 hover:opacity-80 cursor-pointer"
+            />
+            <MdOutlineLocationOn className="text-3xl" />
+            <p className="text-slate-900/80 text-sm"> {place} </p>
+            <div className="relative md:flex">
+              {/* SearchBox */}
+              <Select
+              showSearch
+              placeholder="Select prefecture"
+              optionFilterProp="label"
+              onChange={handleOnChange}
+              options={Prefectures}
+              value={searchParams.get('query')?.toString()}
+              />
+            </div>
+          </section>
+        </div>
+      </nav>
+      <div className="w-min flex flex-row px-20 space-x-8 shadow-sm justify-between items-center">
+      <div className="w-1/2 flex flex-col">
+      <h2 className="text-2xl text-center font-semibold">{todayDate}{" "} {" "}{weatherData?.dt ? convertUnixTimeToDate(weatherData.dt).toLocaleTimeString():null}</h2>
+      <p className="text-2xl text-center font-semibold">Current Weather</p>
+        <div className="w-full bg-white border flex flex-row rounded-xl px-20 space-x-12 shadow-sm justify-between items-center mt-4">
+              <div className='items-center justify-between'>
+              {weatherData?.weather.map(condition =>
+                <div key={condition.id}>
+                  <img src={getIconUrl(condition.icon)} alt={condition.main}/> {condition.main}
+                </div>)
+              }
+              </div>
+              <div className="flex flex-col items-center justify-between">
+                <strong>{Math.floor(weatherData?.main.temp ?? 0)}°</strong>
+                <div>Feels like {Math.floor(weatherData?.main.feels_like ?? 0)}°</div>
+                <div>({Math.floor(weatherData?.main.temp_min ?? 0)}°↓{" "} {" "} {Math.floor(weatherData?.main.temp_max ?? 0)}°↑)</div>
```

Comment:
Instead of `{" "} {" "}` I think flex with gap classnames would be easier to manage.

Reply:
Made the changes